### ISSUE TITLE
Feature request: Stop playing after current track

### ISSUE THREAD

**Issue by MineyMe** [gnumdk/lollypop#45]
Hello

This is a feature request for a preference to stop Lollypop from automatically moving to the next track (but rather just stop playing altogether).

Thanks in advance and keep up the good work!

**Comment by MineyMe** [comment]
Hello again,

I figured I might as well take a stab at this myself instead of sending a flood of bug reports your way (even though you've handled them all very well so far!). The patch can be found here: http://pastebin.com/HVbpbNkp

I have never actually programmed anything in Python but a few very simple scripts, I tried to mimic the current code as much as possible. Please let me know if there's anything wrong with it and I will try to fix it.

Hope this helps!

**Comment by gnumdk** [comment]
Ok, let's talk about the issue :) Your code is ok but i don't see the point of having a global option, why do you need a player that just plays songs one by one?

In fact i plan to just add a contextual menu on back/play/next button with:
- Stop at track end
- Stop at album end
- Stop at artist end

What do you think?

**Comment by MineyMe** [comment]
Thanks for your response. I tend to listen to albums in two ways: sometimes I listen to them entirely, but other times I also tend to just play a few random favorite songs. Another reason is that, when playing an instrument, you sometimes put on a song to play along with it, but whilst the song is finishing and you're still practicing, you don't want to have to stop everything, put aside your instrument, just to stop the player from moving on to the next (sometimes loud) track, and then have to pick up your instrument again, after which the "magic" can sometimes be gone ;).

In short, a global option for me isn't really a requirement and the suggested contextual menu sounds good, as long as it would be possible to have this option permanently applied (i.e. not a "Stop at track end" that you have to reactivate every time you play a new song).

Look forward to your response
